File: C#/begginer/1010.cs
using System;

class URI {

    static void Main(string[] args) { 

        string[] input1 = Console.ReadLine().Split(' ');

        int code1 = int.Parse(input1[0]);
        int units1 = int.Parse(input1[1]);
        double price1 = double.Parse(input1[2]);

        string[] input2 = Console.ReadLine().Split(' ');

        int code2 = int.Parse(input2[0]);
        int units2 = int.Parse(input2[1]);
        double price2 = double.Parse(input2[2]);

        double priceToPay = price1 * units1 + price2 * units2;

        Console.WriteLine($"VALOR A PAGAR: R$ {priceToPay:F2}");

    }

}

File: python/begginer/1010.py
input1 = input().split()
units1 = int(input1[1])
price1 = float(input1[2])

input2 = input().split()
units2 = int(input2[1])
price2 = float(input2[2])

print("VALOR A PAGAR: R$ %.2f" % (units1 * price1 + units2 * price2))
